tiles/monaco.sh

```diff
@@ -0,0 +1,6 @@
+java -Xmx2g -jar target/protomaps-basemap-HEAD-with-deps.jar \
+    --osm_url=https://protomaps.dev/~wipfli/fixtures/monaco-2025-02-15.osm.pbf \
```

Comment:
If this is small enough can we commit it to the repo to make this more self-contained?

If not, maybe the r2-public bucket

Reply:
The file is 128 KB. I am at the moment looking into render tests as well and we might get more pmtiles and expected images. Let's think of this pull request for now as draft...

Reply:
Committed it to git now